tests/functional/src/main/java/org/mule/tck/util/sftp/SftpServer.java

```diff
@@ -89,9 +95,9 @@ public void stop()
     {
         try
         {
-            sshdServer.stop(true);
+            sshdServer.stop(false);
```

Comment:
This does a graceful shutdown of the sftp server instead of an immediate one.
This change didn't have any noticeable time penalty in my testing.